libs/components/src/loading-indicator/loading-indicator.component.ts

```diff
@@ -0,0 +1,7 @@
+import { Component } from "@angular/core";
+
+@Component({
+  selector: "loading-indicator",
```

Comment:
all components have `bit-` prefix, let's make it consistent:
```suggestion
  selector: "bit-loading-indicator",
```